src/pages/workspace/accounting/intacct/advanced/SageIntacctAdvancedPage.tsx

```diff
@@ -101,7 +99,7 @@ function SageIntacctAdvancedPage({policy}: WithPolicyProps) {
         <ConnectionLayout
             displayName={SageIntacctAdvancedPage.displayName}
             headerTitle="workspace.accounting.advanced"
-            headerSubtitle={currentSageIntacctOrganizationName}
+            headerSubtitle={entity}
```

Comment:
This `entity` value now became entity_id after last update, can you update it to entity name please?